HOWTO.md

```diff
@@ -214,17 +272,27 @@ dimensions it is defined on, with a possible position of the step for each
 dimension along whther that step goes up or down (or no step for that axis).
 Evaluation of this function will always return either a 1 or a zero only.
 
-Frankly not really used very much, but it is a non-trivial example of a function
-that cannot be used as a probability density function. And on that note...
-#### 3.2.1.2. PDF
+Frankly not really used very much.
+
+#### 3.2.1.2. 3.2.1.2 SquareRootScale
+`SquareRootScale` is a `Function` whose evaluation function is of the form
+`f(x) = a*sqrt(x)`, where `a` is the "gradient" parameter. This gradient
+can be set directly by `SetGradient(a)`, or indirectly using the standard
+`FitComponent` interface: `SetParameter("grad", a)` etc.
+
+This function is typcially used when modelling an energy resolution systematic: 
+you expect the energy resolution to be proportional to the square root of the energy.
+
+
+#### 3.2.1.3. PDF
 Let's go another level deeper! a `PDF` is *another* abstract class, which is a
 `Function` that also has the property of definite integration, via the 
 `Integrate(mins, maxs)` method. Because a `PDF` has a well-defined integral,
 it can also be also randomly sampled from: `Sample()`. See the section on 
 [random number generation](#4-random-number-generation-rand) for how derived
 classes implement this.
 
-##### 3.2.1.2.1. Gaussian
+##### 3.2.1.3.1. Gaussian
 For all of that effort, we can finally make some actual PDFs, in particular a
 multi-dimensional Gaussian (given how useful they are!). Unsurprisingly, 
 `Gaussian` derives from the `PDF` class.
```

Comment:
on line 305 I think handles -> handled (It wasn't changed in this PR so can't comment on the exact right line)